src/pages/Videos/VideoContent/RelativeVideos/RelativeVideos.jsx

```diff
@@ -0,0 +1,51 @@
+import React, { useEffect, useState } from 'react';
+import { RelatedSection, RelatedBar, RelatedBarTxt, ScrollDiv } from './RelativeVid.styled';
+import VideoCardRelative from '../RelativeVideos/RelativeVideosDetails';
+// import FormatPubDate from '../../utils/formatPublishedDate';
+
+function VideosRelatives({ idVideo }) {

```

Comment:
It is a good practice that the name of your component and the name of your file matches

Reply:
Also I notice inconsistencies in your code. Here you are using `idVideo`, which I love, but in all your code you are using `props`. You need to have a style in the whole app. Either passing props and deconstructing right after, or deconstructing as much as you can. Hope makes sense.